static/index.js

```diff
@@ -91,4 +91,50 @@ document.addEventListener("DOMContentLoaded", event => {
     }
   });
 
+  var darkSwitch = document.querySelector('input[name=darkMode]');
+  const prefersDarkScheme = window.matchMedia("(prefers-color-scheme: dark)");
+  const prefersLightScheme = window.matchMedia("(prefers-color-scheme: light)");
+  const _darkModeLocal = localStorage.getItem("darkModeLocal");
```

Comment:
Since we are changing the value of local storage later, let's not store the value. I originally meant to store `darkModeLocal` as const.

```js
const appThemeLocalKey = "appThemeLocal";
// and then later throughout the code, use like this
if (localStorage.getItem(appThemeLocalKey)) {
    // ...
}
```